libs/vre/resource-editor/resource-editor/src/lib/resource-info-bar.component.ts

```diff
@@ -19,9 +19,9 @@ import { filter, map, take } from 'rxjs/operators';
         <span *ngIf="resourceAttachedUser"
           >by
           {{
-            resourceAttachedUser.username
-              ? resourceAttachedUser.username
-              : resourceAttachedUser.givenName + ' ' + resourceAttachedUser.familyName
+            resourceAttachedUser.givenName || resourceAttachedUser?.familyName
```

Comment:
I bet both `givenName` and `familyName` should be both presented.
```suggestion
            resourceAttachedUser?.givenName ?? resourceAttachedUser?.familyName
```

Reply:
@mpro7 Yes, I tested it manually. The missing fields will not be displayed, only the present names. If both are missing, the username will be displayed.